// @expo-google-fonts/saira/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Saira_100Thin = require('./100Thin/Saira_100Thin.ttf');
export const Saira_200ExtraLight = require('./200ExtraLight/Saira_200ExtraLight.ttf');
export const Saira_300Light = require('./300Light/Saira_300Light.ttf');
export const Saira_400Regular = require('./400Regular/Saira_400Regular.ttf');
export const Saira_500Medium = require('./500Medium/Saira_500Medium.ttf');
export const Saira_600SemiBold = require('./600SemiBold/Saira_600SemiBold.ttf');
export const Saira_700Bold = require('./700Bold/Saira_700Bold.ttf');
export const Saira_800ExtraBold = require('./800ExtraBold/Saira_800ExtraBold.ttf');
export const Saira_900Black = require('./900Black/Saira_900Black.ttf');
export const Saira_100Thin_Italic = require('./100Thin_Italic/Saira_100Thin_Italic.ttf');
export const Saira_200ExtraLight_Italic = require('./200ExtraLight_Italic/Saira_200ExtraLight_Italic.ttf');
export const Saira_300Light_Italic = require('./300Light_Italic/Saira_300Light_Italic.ttf');
export const Saira_400Regular_Italic = require('./400Regular_Italic/Saira_400Regular_Italic.ttf');
export const Saira_500Medium_Italic = require('./500Medium_Italic/Saira_500Medium_Italic.ttf');
export const Saira_600SemiBold_Italic = require('./600SemiBold_Italic/Saira_600SemiBold_Italic.ttf');
export const Saira_700Bold_Italic = require('./700Bold_Italic/Saira_700Bold_Italic.ttf');
export const Saira_800ExtraBold_Italic = require('./800ExtraBold_Italic/Saira_800ExtraBold_Italic.ttf');
export const Saira_900Black_Italic = require('./900Black_Italic/Saira_900Black_Italic.ttf');
